### ISSUE TITLE
Image for Element.getBoundingClientRect() Documentation appears wrong or confusing

### ISSUE THREAD

**Issue by bshaibu** [mdn/sprints#2965]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [X] Correction or update

# Details
Link: https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect
Image in question:
![image](https://user-images.githubusercontent.com/2855159/76718524-49892700-66f4-11ea-95f5-93940390572e.png)

The current image for `Element.getBoundingClientRect()` is fairly hard to parse if not outright incorrect.

**Expected**
I would expect the blue shape to be the client rectangle being described. I would also expect the top label to be on the line indicating the top of the DOMRect, the bottom at the bottom, and so on.

**Actual**
The current image (and accompanying description) doesn't make it clear what rectangle is being described and seems to have top-bottom swapped with left-right if I'm reading it correctly.

**Comment by chrisdavidmills** [comment]
Hi there @bshaibu !

I've had a look at this, and the image seems correct to me, although it really could've done with a bit more explanation.

To hopefully help with this, I've added some more description to the top fo the article, and added an example to more effectively illustrate what the values mean.

Does this help?

**Comment by bshaibu** [comment]
Hi @chrisdavidmills , thank you for the quick reply and update! The example is helpful and exactly matches my understanding and testing of `Element.getBoundingClientRect()` and `DOMRect`s in general.

To be honest, I still find the image a bit confusing, but I see what my misunderstanding was now.  I was interpreting the image as if it was saying that Left, Top, Right, and Bottom were signified by their closest line.
Please excuse my terrible web Paint here, but I was interpreting the image as:
![getBoundingClientRectangles_misunderstanding](https://user-images.githubusercontent.com/2855159/76764527-39a03000-6752-11ea-8bc2-81f3ffd8cc9b.png)
which didn't align with my expectations.

I realize now that the lines are actually indicating that each respective position is that far from the origin using a line and an additional "serif" line. 

It might be more clear if the lines were:

- color-coded (another bad paint example)
![getBoundingClientRectanglesColored](https://user-images.githubusercontent.com/2855159/76765241-4ec98e80-6753-11ea-809d-f7c938b60fd3.png)
- shifted so they don't have left and top overlapping (similar to how bottom and right are currently)
![getBoundingClientRectanglesShifted](https://user-images.githubusercontent.com/2855159/76766478-483c1680-6755-11ea-857c-28ee7d5e8c7f.png)
- or if there was a line in the Value section saying something like "In the image to the right, `top` is measured as the distance from the top of the viewport to blue rectangle. Similarly, `left` is measured as the distance from the left of the viewport. Note top, left, bottom, and right are coordinates values."

This issue should be fine to close out as a user error (this was my misinterpretation!), but just wanted to share some things for you all to consider.

**Comment by chrisdavidmills** [comment]
Hey @bshaibu !

I totally get what you mean about the diagram being confusing now. I've replaced it with a hopefully better one: https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

Let me know what you think!

**Comment by bshaibu** [comment]
Thank you for all the updates here @chrisdavidmills! The new image is a lot clearer on where the arrows and labels match up.

I am a little worried that now the image implies that `bottom` is the distance from the bottom of the viewport to bottom of the rectangle (and similar for `right`), but I've nitpicked on this enough already. Thanks again!

**Comment by chrisdavidmills** [comment]
@bshaibu no, no, you are right to nitpick. My new diagram was wrong ;-)

I have update it to another new diagram. Have I got it right this time?

**Comment by bshaibu** [comment]
@chrisdavidmills I think you've hit the sweet spot - super clear diagram and correct across the board. Thank you for getting this all the way!

**Comment by chrisdavidmills** [comment]
@bshaibu woo hoo!

And thank you for the patient, careful analysis of the problem, which led to some fun collaboration.
